### ISSUE TITLE
Simplifying the Loss: Can (1 - t) Be Dropped?

### ISSUE THREAD

**Issue by ranrhuang** [LTH14/JiT#60]
Hi, thanks for this excellent work!

I have a question regarding the loss formulation. Now we define:
```
v = (x - z) / (1 - t).clamp_min(self.t_eps)
v_pred = (x_pred - z) / (1 - t).clamp_min(self.t_eps)
loss = (v - v_pred) ** 2
```
Under this formulation, can the loss be simplified to: `(x - x_pred)**2`, i.e., effectively dropping the (1 - t)**2 term in the denominator?

Or does the (1 - t) scaling play an essential role (e.g., as a time-dependent weighting) that makes this simplification invalid?

**Comment by swimmincatt35** [comment]
I will try to answer this question to my best ability. The paper is inspiring and the codebase is very simple to follow. Huge thanks to the authors!

I agree scaling plays a crucial role here, and to see how scaling affects the training results, I believe you can refer to Table 2a and 2b. The authors showed that under different scaling schemes in lower-d diffusion training, the velocity reparametrization showed to be the best among all three modeling cases. And you are right, it is effectively scaling by (1-t)**2. In the case of removing this scaling, the FID reads as 5.76 on ImageNet64 and 10.14 on ImageNet256, both worse than adding the scaling / reparametrizing to predicting v.

**Comment by ranrhuang** [comment]
Thanks for your help. However, I noticed that the authors mention in their experiments that the loss weighting induced by the v-loss is preferable, but not critical. This suggests that it may not be strictly necessary.
